module/avl/avl.c

```diff
@@ -1008,7 +1008,7 @@ avl_destroy_nodes(avl_tree_t *tree, void **cookie)
 	--tree->avl_numnodes;
 
 	/*
-	 * If we just did a right child or there isn't one, go up to parent.
+	 * If we just made a right child or there isn't one, go up to parent.
```

Comment:
I don't think we are "making" any nodes here; "did" seems ok to me, or we could change it to "removed"